_posts/2021-01-13-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,42 @@
+
+### - Goals of Audio Mixing
+
+The goals of audio mixing can be as simple as taking a microphone and game audio then pushing it to a single output. They can also be as elaborate as dividing out a microphone, or even two or three microphones, game audio, alerts, computer sounds, and music. Each having their own source to pull audio from then pushing that audio to two or more outputs, headphones, broadcast software such as OBS or SLOBS, or discord. Of course we don't want our stream audio going to friends on discord, and we don't want our mic audio going to our speakers if we aren't using headphones at the moment.
+
+This is where audio mixing software and hardware comes into play. Audio mixing allows us to properly route our audio to where it needs to be and often times gives it a boost in quality, while also making it easier to handle unlike windows sound settings and such. Having more options can be daunting at times, especially when just starting out, but once you get the hang of it, more often than not it saves so much more time and provides you with better quality than if you just went without.
+
+### - Audio Mixing options
+
+Are you interested in having more control over your audio? Lets go over our options then!
+
+###### - Mixing in OBS?
+You might say OBS has its own audio mix, that is true. However, just like using windows, it can be complicated. First you have to find audio levels in windows that suite yourself, so you can hear the game and other things. Then you have to pull the audio directly into OBS and try and manage it there. It can be a complete mess because the audio levels are already being adjusted for your hearing first. Optionally, if you adjust for the OBS Mix first, the levels for you hearing the audio will also be affected, too high, too low, and when you adjust the audio levels going to your headphones it will then adjust the levels in OBS. Big wonky mess right?
+
+###### - GoXLR, GoXLR-Mini, Elgato Wave
+ I'm sure every streamer has heard of a GoXLR, or GoXLR-Mini which come with software mixing alongside hardware controls. There is also the Elgato Wave microphones which come with similar software mixing and some hardware controls.
+
+###### - Physical Mixers
+Physical mixers are also available of various sizes and prices that stretch from less than $50 US Dollars to well over $500 to even $1000s. These beastly devices offer physical nobs and sliders for customization and take up a bit more space than a GoXLR but in some cases not by much or at all as some of them can be fairly small. However, their prices range from significantly lower than even a GoXLR-Mini or even cheaper then the Elgato Wave.
```

Comment:
May mention that the average price for a mixer that's plenty capable for a streamer is around $200.

Reply:
I would think this is more than sufficient to get started?: https://www.amazon.com/BEHRINGER-XENYX-1202FX-3-pin-Black/dp/B000J5Y214/ref=psdc_11974871_t1_B00IBIVL42
[4:50 PM]
its basically a baby version of what I have.

Reply:
It has XLR Input with toggle-able phantom power, multiple line inputs, a main mix output and a separate FX Send for the microphone for discord.  I would use it as a baseline/entry-level physical mixer for minimum requirement almost.

Reply:
made a reference to: https://www.amazon.com/Behringer-Q1202USB-12-Channel-Mixer/dp/B008O517IW/ref=pd_di_sccai_7?pd_rd_w=BfNVF&pf_rd_p=c9443270-b914-4430-a90b-72e3e7e784e0&pf_rd_r=2RGC09XGTQEZ1NBV91TR&pd_rd_r=3b2735af-b608-4861-8b7e-204b93464022&pd_rd_wg=7SIjc&pd_rd_i=B008O517IW&psc=1 's price instead since it covers a compressor as well. Previous mixer didn't ahve compressor ; ;